### DISCUSSION TITLE
モバイル学習コンテンツで、各プラットフォーム（Android、iOSアプリ）の差異を意識しなくてよい旨を記載したい

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#259] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/basic-concepts/react-native-basics
React Nativeの基本

React Nativeが各プラットフォーム（Android、iOSアプリ）の差異を吸収してくれて、
各プラットフォームを意識せずに開発できる。ということを記載したい。

例えば、テキストを表示する場合、Androidでは<ViewGroup>、IOSでは<UIView>
だが、React Nativeを利用すれば<Text>と定義すればよい。

**Comment by nagano-ryo** [comment]
@tani-takanori さんより以下のコメントあり。
すべて意識しなくて済むわけではない。
「完全に各プラットフォームを意識せずに開発できる。」は言い過ぎで誤解を招いてしまう。

**Comment by creature-water-valley** [comment]
以下のPRで対応済み
- https://github.com/ws-4020/mobile-app-crib-notes/pull/1294/files#diff-97c64833fe086fe565015037e6547f0e7b19c214af259ab854ab4c0ed3858bcbR18
